conf/upf.json

```diff
@@ -56,6 +56,8 @@
 
     "": "Whether to enable P4Runtime feature",
     "enable_p4rt": false,
+    "" : "conn_timeout: 1000",
```

Comment:
why not "conn_timeout": 1000 ?? why that "" ??

Reply:
that's commented out.. we will configure it in case we use some different value